zerver/openapi/python_examples.py

```diff
@@ -216,6 +216,79 @@ def get_members(client):
         else:
             assert member.get('profile_data', None) is not None
 
+@openapi_test_function("/users/{user_id}:get")
+def get_single_user(client):
+    # type: (Client) -> None
+
+    # {code_example|start}
+    # Get user data of a user
+    user_id = 8
+    url = 'users/' + str(user_id)
+    result = client.call_endpoint(
+        url=url,
+        method='GET'
+    )
+    # {code_example|end}
+    validate_against_openapi_schema(result, '/users/{user_id}', 'get', '200')
+
+    # {code_example|start}
+    # You may pass the `client_gravatar` query parameter as follows:
+    result = client.call_endpoint(
+        url=url,
+        method='GET',
+        request={'client_gravatar': True}
+    )
+    # {code_example|end}
+
+    validate_against_openapi_schema(result, '/users/{user_id}', 'get', '200')
+    assert result['members'][0]['avatar_url'] is None
+
+    # {code_example|start}
+    # You may pass the `include_custom_profile_fields` query parameter as follows:
```

Comment:
This should explain the use case.
"If you'd like data on custom profile fields, you can request them as follows"

Reply:
Though also reading this documentation, maybe we should just make the single-user endpoint always include custom profile fields?  That might be a better API; I don't see the use case for a "slimmer" version that doesn't fetch that data as well when you're just fetching a single user.

Reply:
@timabbott I think I should open a PR for things that require change in the view code after this gets merged. I'll also make changes in the documentation accordingly.

Reply:
Or I can make a PR doing that (which gets merged before this) and amend the commits in this PR.

Reply:
A follow-up PR is fine.